x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -93,6 +93,11 @@ interface BaseProps {
    * Order for keyboard dragging. This takes an array of numbers which will be used to order hierarchically
    */
   order: number[];
+
+  /**
+   * don't display ghost image for the drop element
+   */
+  noGhost?: boolean;
```

Comment:
I find negative props somewhat confusing in components. Maybe others do to?

Especially when passing in `false` you end up with a double negative when thinking it through and that can throw people (me, at least) for a loop.

Reply:
I understand and it make sense. However, on the other side, this construction allows us to treat displaying a ghost as a default behavior without passing extra parameter and that's what I was trying to do here. So for most components I can do `<DragDrop draggable .../>` and I don't have to add `ghost = {true}` (or specify a default value in the component). I understand it's a bit on the edge when it comes to readability so I'd like to know the opinion of another person reviewing this 🙏